xbmc/cores/VideoPlayer/AudioSinkAE.h

```diff
@@ -74,7 +74,7 @@ class CAudioSinkAE : IAEClockCallback
   double m_timeOfPts;
   double m_syncError;
   unsigned int m_syncErrorTime;
-  double m_resampleRatio;
+  double m_resampleRatio = 0.0;
```

Comment:
0.0 indicates an invalid state. I couldn't find enough uses of this variable in this class to make a better guess.